README.md

```diff
@@ -0,0 +1,34 @@
+# Docker images for opensnp.org
+
+Currently two images - a *opensnp.org-docker* image and a *dev* image. The *opensnp.org-docker* image in the root folder pulls all gems and other dependencies. The *dev* image adds a few dummy files so that you can run the image in a local container.
+
+# How to build the images
+
+In a screen session:
```

Comment:
I see your point.
You are not supposed to step into the containers and, least of all, run anything manually in there.